templates/advanced/src/tags/components/example/example.js

```diff
@@ -0,0 +1,8 @@
+import { tag } from '@storefront/core';
+
+@tag('example-component', require('./example.html'), require('./example.css'))
```

Comment:
The `html` file is not really necessary (it's just more files for them to delete), you can just put something inline here. Eg, `'<span></span>'`. Same thing with the `css`

Reply:
@Vic-Dev I thought this would be useful as it demonstrates the connection between a given component and the files which must be mocked in order to test it (ie. in `bootstrap.js`). That said, happy to remove if you don't think it's worth it?

Reply:
I agree with @jrmykolyn in that having the html file helps to show someone all the files that they need to mock in order to get component testing up and running.